includes/payment-methods/class-upe-split-payment-gateway.php

```diff
@@ -93,6 +94,20 @@ public function __construct(
 			$this->id           = self::GATEWAY_ID . '_' . $this->stripe_id;
 			$this->method_title = "WooCommerce Payments ($this->title)";
 		}
+
+		add_filter( 'woocommerce_available_payment_gateways', [ $this, 'add_sepa_gateway' ], 100, 1 );
```

Comment:
`100` was just a random high number to make sure the callback is being executed as the last one - feel free to improve on this